Source/WebKit/UIProcess/API/Cocoa/WKMenuItemIdentifiersPrivate.h

```diff
@@ -68,5 +68,6 @@ WK_EXTERN NSString * const _WKMenuItemIdentifierCheckSpellingWhileTyping WK_API_
 WK_EXTERN NSString * const _WKMenuItemIdentifierCheckGrammarWithSpelling WK_API_AVAILABLE(macos(13.0), ios(16.0));
 WK_EXTERN NSString * const _WKMenuItemIdentifierPlayAllAnimations WK_API_AVAILABLE(macos(13.3), ios(16.4));
 WK_EXTERN NSString * const _WKMenuItemIdentifierPauseAllAnimations WK_API_AVAILABLE(macos(13.3), ios(16.4));
+WK_EXTERN NSString * const _WKMenuItemIdentifierToggleShowSubtitles WK_API_AVAILABLE(macos(14.0), ios(17.0));
```

Comment:
This won't actually be available in macOS 14 / iOS 17, so you should use `WK_MAC_TBA` and `WK_IOS_TBA` here.